previsionio/plotter.py

```diff
@@ -298,9 +301,9 @@ def plot_conf_mat(self, predict_id):
             predict_id (str): ID of the prediction
 
         """
-        if self.usecase.type_problem != 'multiclassification':
-            raise Exception('Confusion matrices only available for multiclassification, not {}.'.format(
-                self.usecase.type_problem))
+        if self.usecase.training_type != 'multiclassification':
```

Comment:
Could be:

`if self.usecase.training_type != TypeProblem.MultiClassification:`